partitioned-pipe-two-phase/monolithic/clean.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -e -u
+
+. ../../tools/cleaning-tools.sh
+
+clean_openfoam .
```

Comment:
```suggestion
clean_openfoam .
rm -r 0
```

I guess we also need this here.